docs/_steps/adding_editing_workflow_step.md

```diff
@@ -0,0 +1,65 @@
+---
+title: Adding or editing workflow steps
+lang: en
+slug: adding-editing-steps
+order: 3
+---
+
+<div class='section-content'>
+
+When a builder adds (or later edits) your step in their workflow, your app will receive a `workflow_step_edit` action. The callback assigned to the `edit` property of the `WorkflowStep` configuration object passed in during instantiation will run when this action occurs.
+
+Whether a builder is adding or editing a step, you need to provide them with a special `workflow_step` modal — a workflow step configuration modal — where step-specific settings are chosen. Since the purpose of this modal is tied to a workflow step's configuration, it has more restrictions than typical modals—most notably, you cannot include `title​`, `submit​`, or `close`​ properties in the payload. By default, the `callback_id` used for this modal will be the same as that of the workflow step.
```

Comment:
```suggestion
Whether a builder is adding or editing a step, you need to send them a [workflow step configuration modal](https://api.slack.com/reference/workflows/configuration-view) with a type of `workflow_step`. This modal is where step-specific settings are chosen, and it has more restrictions than typical modals—most notably, it cannot include `title​`, `submit​`, or `close`​ properties. By default, the configuration modal's `callback_id` will be the same as the workflow step.
```

Reply:
you could remove "with a type of `workflow_step`" since that's included by default. but i thought it was kinda helpful that you included it.